kolibri/plugins/demo_server/assets/src/DemoServerBannerContent.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+
+  <h1 class='banner-header' v-if="bannerClosed">
+    {{ $tr('demoServerHeader') }}
+  </h1>
+  <div v-else>
+    <h1>{{ $tr('demoServerHeader') }}</h1>
+    <p>{{ $tr('demoServerP1') }}</p>
+    <ul>
+      <li>{{ $tr('demoServerL1', {user: 'learnerdemo', pass: 'pass'} ) }}</li>
+      <li>{{ $tr('demoServerL2', {user: 'coachdemo', pass: 'pass'} ) }}</li>
+      <li>{{ $tr('demoServerL3', {user: 'admindemo', pass: 'pass'} ) }}</li>
+    </ul>
+    <p>{{ $tr('demoServerP2') }}</p>
+    <p>{{ $tr('demoServerP3') }}</p>
+    <ul>
+      <li>
+        <KExternalLink
+          href="https://learningequality.org/documentation/"
+          :text="$tr('demoServerA1')"
+          target="_blank"
+        />
+      </li>
+      <li>
+        <KExternalLink
+          href="https://learningequality.org/download/"
+          :text="$tr('demoServerA2')"
+          target="_blank"
+        />
+      </li>
+    </ul>
+  </div>
+
+</template>
+
+
+<script>
+
+  import KExternalLink from 'kolibri.coreVue.components.KExternalLink';
+  import themeMixin from 'kolibri.coreVue.mixins.themeMixin';
+
+  export default {
+    name: 'DemoServerBannerContent',
+    components: { KExternalLink },
+    mixins: [themeMixin],
+    props: {
+      bannerClosed: {
+        required: true,
+        type: Boolean,
+        default: false,
+      },
+    },
+    $trs: {
+      demoServerHeader: 'Welcome to the Kolibri demo site!',
+      demoServerP1: 'Explore any of the three primary user types:',
+      demoServerL1: 'Learner ({user}/{pass} or access as a guest)',
+      demoServerL2: 'Coach ({user}/{pass})',
+      demoServerL3: 'Admin ({user}/{pass})',
+      demoServerP2:
+        'This online demo is intended for demonstration purposes and any user may be periodically cleared. This demo has features of the latest Kolibri version and all resources found are samples.',
+      demoServerP3:
+        'To learn more about using Kolibri in an offline context and better understand the platform:',
+      demoServerA1: 'Read the documentation',
+      demoServerA2: 'Download and install the latest release',
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .close-button {
+    float: right;
+    margin-bottom: 24px;
+  }
+  .open-button {
+    float: right;
+    margin-top: 0;
+  }
+  .banner-header {
+    float: left;
+  }
```

Comment:
When there are horizontal layouts, we should specify grid behaviors for the different screen sizes (cc @khangmach).

(Floats tend to cause issues when adjacent items collide with each other, while the grids keep everything nice and consistent.)

For example, the spec might say...

> **large size - max-width: 1000px centered**
> 
> ```
> --------------------------------------------------------------------------
> | title: 10/12 cols left-aligned                 |     button: 2/10 cols |
> |                                                |         right-aligned |
> --------------------------------------------------------------------------
> ```
> 
> **medium size**
> ```
> -------------------------------------------------------------
> | title: 6/8 cols left-aligned         |   button: 2/8 cols |
> |                                      |      right-aligned |
> -------------------------------------------------------------
> ```
> 
> **small size**
> ```
> ---------------------------------------
> | title: 4/4 cols left-aligned        |
> ---------------------------------------
> | button: 4/4 cols left-aligned       |
> ---------------------------------------
> ```

And this could be implemented with (roughly):

```html
<KGrid>
  <KGridItem sizes="4, 6, 10">
    {{ title }}
  </KGridItem>
  <KGridItem sizes="4, 2, 2">
    <KButton />
  </KGridItem>
</KGrid>
```

(Unfortunately that makes it a bit harder to drop in the Content component as you have implemented here.)

Reply:
Thanks Khang! I think I can work this out using the KGrid